.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI
+
+on:
+- push
```

Comment:
`- pull_request` is missing.

Reply:
Oh, I'll add it directly to the master branch! Thanks!